Lesson_07/homework_7.py

```diff
@@ -0,0 +1,217 @@
+# task 1
+""" Задача - надрукувати табличку множення на задане число, але
+лише до максимального значення для добутку - 25.
+Код майже готовий, треба знайти помилки та виправити\доповнити.
+"""
+from typing import List
+
+
+def multiplication_table(number):
+    # Initialize the appropriate variable
+    multiplier = 1
+
+    # Complete the while loop condition.
+    while multiplier <= number:
+        result = number * multiplier
+
+        if result > 25:
+            # Enter the action to take if the result is greater than 25
+            pass
```

Comment:
The idea here to go out from the while loop immediately if the result is more than 23. You leave just pass. Please fix this part.